src/docfx/lib/markdown/LinkExtension.cs

```diff
@@ -36,6 +39,13 @@ internal static class LinkExtension
                     {
                         inline.Tag = ResolveLinks(inline.Tag, inline);
                     }
+                    else if (node is TripleColonBlock tripleColonBlock && tripleColonBlock.Extension is ImageExtension)
+                    {
+                        var imageSrc = tripleColonBlock.GetAttributes().Properties.Where(kv => kv.Key == "src").FirstOrDefault().Value;
```

Comment:
`.FirstOrDefault()` is there always a "src" property? otherwise, it could throw NRE
If always, then we should use `tripleColonBlock.GetAttributes().Properties.Single(kv => kv.Key == "src").Value`

Reply:
yes there will always a "src" property